### PR TITLE
Add 14 websites to the global dark list

### PR DESCRIPTION
**Pull request by KaKi87**
Thanks

### CODE REVIEW

File: src/config/dark-sites.config [hunk 1]
```diff
@@ -222,6 +226,7 @@ disneyplus.com
 distro.tube
 djflame.tech
 dlive.tv
+doc.rust-lang.org/book/
```
Comment (Gusted) [inline src/config/dark-sites.config:229]: Please remove, this isn't dark-by-default:

image

File: src/config/dark-sites.config [hunk 2]
```diff
@@ -765,6 +778,7 @@ tf2mart.net
 tgw1916.net/bacteria_abis.html
 thealiendrew.github.io
 thelinuxcast.org
+theme-park.dev
```
[the remark this answers is not in the document]
Comment (KaKi87) [reply src/config/dark-sites.config:781]: No, why ?

File: src/config/dark-sites.config (hunk 1 above)
Comment (KaKi87) [reply src/config/dark-sites.config:229]: Oh, my bad, I must have forgot to disable Dark Reader while performing the test :sweat_smile:

File: src/config/dark-sites.config (hunk 2 above)
[the remark this answers is not in the document]
Comment (Gusted) [reply src/config/dark-sites.config:780]: Seems like it was redirecting for me, weird...

**Review by Gusted — APPROVED** [review]
Thanks.

**Pull request merged**